<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Package</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>591</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="line_edit_etalon_traces">
               <property name="text">
                <string>Эталонные трассы ВО</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>Table G-T_rls</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Table T-S</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Save to file G-T_rls</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Save to file T-S</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>SK42</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WGS84</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>PZ90_11</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="line_edit_array_coord_points">
               <property name="text">
                <string>Массив КТ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_4">
               <item>
                <property name="text">
                 <string>Table S-T</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Save to file S-T</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>521</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Перепаковка данных во внутренний формат программы</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>461</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>310</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Отображение данных в таблице</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>780</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Тип конвертации</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>810</y>
          <width>741</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Arimo&quot;;</string>
        </property>
        <property name="text">
         <string>Example</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="geometry">
         <rect>
          <x>485</x>
          <y>411</y>
          <width>481</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>390</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>POI output</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>380</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Etalon traces output</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>0</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;Arimo&quot;;</string>
        </property>
        <property name="text">
         <string>Statuses</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>130</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>230</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear all data</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>50</y>
          <width>301</width>
          <height>74</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>etalon_data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>poi_data</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit_18"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_19"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Visualization</string>
        </property>
       </widget>
       <widget class="QCustomPlot" name="plot" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>561</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_evaluation">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>540</y>
          <width>101</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Evaluation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>540</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>--- Чтобы увидеть результат оценки
Перейдите во вкладку Evaluation</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>40</y>
          <width>341</width>
          <height>218</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Отображенные координаты (правой кнопкой мыши)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_2">
           <column>
            <property name="text">
             <string>x</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>y</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="draw_etalon_data">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>430</y>
          <width>121</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Draw etalon data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="draw_poi_data">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>430</y>
          <width>121</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Draw poi data</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>530</y>
          <width>280</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;Arimo&quot;;</string>
             </property>
             <property name="text">
              <string>Etalon Trace</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;Arimo&quot;;</string>
             </property>
             <property name="text">
              <string>POI trace</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="lineEdit_20"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_21"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Evaluation</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Arimo&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Точностные характеристики</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>981</width>
          <height>271</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>310</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;Arimo&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Оценка дальностей обнаружения</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>981</width>
          <height>261</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>631</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Дальность, м</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Азимут, рад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Угол места, рад</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_11"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Радиальная скорость, м/с</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_12"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>141</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="styleSheet">
            <string notr="true">;</string>
           </property>
           <property name="text">
            <string>Полная ошибка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Сист. ошибка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>СКО</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>801</width>
          <height>76</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Минимальная дальность обнаружения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_13"/>
           </item>
           <item>
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>м, при высоте полета цели</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_14"/>
           </item>
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>м</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>Максимальная дальность обнаружения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_15"/>
           </item>
           <item>
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>м, при высоте полета цели</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_16"/>
           </item>
           <item>
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>м</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>490</y>
          <width>431</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>Коэффициент реализации радиогоризонта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_17"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
   <slots>
    <signal>onSelectionChanged()</signal>
    <slot>selectionChangedByUser()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
